### PR TITLE
docs(performance-tips): added steps to deploy io engine deamonset

### PR DESCRIPTION
**Pull request by AJDatacore**

### CODE REVIEW

File: quickstart/performance-tips.md [hunk 1]
```diff
@@ -93,17 +93,46 @@ cat /sys/devices/system/cpu/isolated
 {% endtab %}
 {% endtabs %}
 
-### Deploy Mayastor daemonset
+### Deploy IO-engine daemonset
```
Comment (tiagolobocastro) [inline quickstart/performance-tips.md]: There's no deploy of the daemonset now, it's all done via helm or kubectl edit.
Should we also show the helm command to change the coreList? CC @niladrih 

```suggestion
### Modify the io-engine daemonset
```

Comment (niladrih) [reply quickstart/performance-tips.md]: The core list set command for installation and upgrade would be a good idea. I'll test this one out, and post here.

Comment (niladrih) [reply quickstart/performance-tips.md]: Install command argument: `--set 'io_engine.coreList={30,31}'`
Upgrade command argument: `--set-args 'io_engine.coreList={30,31}'`

Need to have the single quotes around it.

File: quickstart/performance-tips.md [hunk 2]
```diff
@@ -93,17 +93,16 @@ cat /sys/devices/system/cpu/isolated
...
-    - name: mayastor
-      ...
-      args:
-        ...
-        - "-l3,4"
+### Update mayastor helm chart for CPU core specification
+
+To allot specific CPU cores for Mayastor's reactors, follow these steps:
+
+1. Ensure that you have the Mayastor kubectl plugin installed, matching the version of your Mayastor Helm chart deployment ([releases](https://github.com/openebs/mayastor/releases)). You can find installation instructions in the [Mayastor kubectl plugin documentation](https://github.com/openebs/mayastor-extensions/blob/develop/k8s/plugin/README.md).
```
Comment (niladrih) [inline quickstart/performance-tips.md]: Let's replace the REAME.md link with this one -- https://mayastor.gitbook.io/introduction/advanced-operations/kubectl-plugin

On second thought, that one isn't very helpful for a naive user.

**Review by niladrih — COMMENTED** [review]
Left one comment.

**Review by niladrih — APPROVED** [review]
Looks good

**Pull request merged**